ReactQt/runtime/src/reactuimanager.cpp

```diff
@@ -128,8 +128,8 @@ void ReactUIManager::setChildren
 {
   //TODO: This is a simple implementation which fixes a broken example. It's not properly tested and may need revisiting
   QList<int> indices;
-  foreach(int i, childrenTags) {
-    indices.append(0); //0 is a default z-order in qml
+  for(int i = 0; i < childrenTags.size(); ++i) {
+    indices.append(i); //Incrementing per each child. Similar to iOS. See the comment in manageChildren
```

Comment:
hi @MaxRis, 
thanks for the comment. I agree that setting z order ourselves (without having that set intentionally in js ) seems not natural and possibly can bring to some unexpected cases (though in my understanding it has pretty low chance). 
I tried to follow/continue the logic of existing  `manageChildren` method, where the indices of children for setting into parent was used directly to set Z order. And if I get correctly, the comment there seems to indicate that - "//on iOS, order of the subviews implies z-order, implicitly its the same in QML". 
I can remove setting Z order or set it to 0 as suggested.